tslint.json

```diff
@@ -40,7 +45,12 @@
                 "variable-declaration": "nospace"
             }
         ],
+        "linebreak-style": {"options": ["CRLF"]},
```

Comment:
Make the Mac guy go CRLF huh? :-)

Reply:
Lol. The files were already CRLF, so... :) This was picked up from our internal extension's settings.

---
In reply to: [407448772](https://github.com/microsoft/sarif-vscode-extension/pull/235#discussion_r407448772) [](ancestors = 407448772)